runtimes/mlflow/mlserver_mlflow/runtime.py

```diff
@@ -94,11 +95,26 @@ async def load(self) -> bool:
         # TODO: Log info message
         model_uri = await get_model_uri(self._settings)
         self._model = mlflow.pyfunc.load_model(model_uri)
+
         self._input_schema = self._model.metadata.get_input_schema()
+        self._signature = self._model.metadata.signature
+        self._sync_metadata()
 
         self.ready = True
         return self.ready
 
+    def _sync_metadata(self):
```

Comment:
I generally like to have a return type `None` in these cases just to make it explicit that we didnt forget about return types, but just a slight preference and I think your still is also valid.

Reply:
Do you mean as part of the type hints? Or on the `return` line?

Reply:
as a type hint ie. `def _sync_metadata(self) -> None:`